randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1366,11 +1372,17 @@ Extra - asset_id: collision_camera_011
   * Extra - start_point_actor_name: PRP_CV_EnergyStation002_WeightPlate
   * Extra - start_point_actor_def: actordef:actors/props/weightactivatedplatform_energy/charclasses/weightactivatedplatform_energy.bmsad
   > Event - Early SA Shortcut Blob
-      Morph Ball and Wave Beam
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Wave Beam
+              Slide and Pseudo-Wave Beam (Beginner) and Slide Jump (Beginner)
   > Door to Varia Suit Tutorial South
       All of the following:
           After Artaria - Thermal Device
-          Space Jump or Simple IBJ
+          Any of the following:
+              Space Jump or Simple IBJ
+              Walljump (Intermediate) and Use Spin Boost
```

Comment:
Gravity + Speed Boster also possible here

Reply:
Can you actually get up to the energy recharge with the speed still?

Reply:
Yeah it's enough I tested it.

Reply:
how tight is it?